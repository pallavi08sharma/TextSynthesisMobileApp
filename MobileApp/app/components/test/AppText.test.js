import React from 'react';
import renderer from 'react-test-renderer';

import AppText from '../AppText';

describe('<AppText />', () => {
    it('renders correctly', () => {
      const tree = renderer.create(<AppText />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });