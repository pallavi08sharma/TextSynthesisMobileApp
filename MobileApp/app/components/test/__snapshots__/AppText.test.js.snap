// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppText /> renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#0c0c0c",
        "fontFamily": "Avenir",
        "fontSize": 18,
      },
      undefined,
    ]
  }
/>
`;
